images/grafana/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if [ -z "$IMAGE" ]; then
+    GRAFANA_IMAGE="quay.io/rh-jmc-team/container-jfr-grafana-dashboard"
+fi
+
+if [ -z "$TAG" ]; then
+    GRAFANA_TAG="0.1.0"
+fi
```

Comment:
I think the variables in the if-conditions should also be prefixed with `GRAFANA_` here. This would allow us to do `GRAFANA_IMAGE=blah make images`.